civictechprojects/static/css/partials/_SplashScreen.scss

```diff
@@ -11,7 +11,7 @@
 
 .SplashScreen-fill {
   min-height: calc(
-    100vh - #{$header-height} - #{$footer-height}
+    100vh - 100px
```

Comment:
This is a bit out of scope for the PR but since I was in SplashScreen anyway I searched the code, saw we had two identical variables used nowhere else, and simplified. 

I expect when the nav changes go in we'll want to revist this number (some rough tests suggest the lower the better, down to a point -- say, 30-50px) but for now I just cleaned up the math, no changes in end result.